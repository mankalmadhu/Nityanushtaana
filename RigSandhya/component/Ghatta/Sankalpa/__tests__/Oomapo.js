import React from 'react';
import ಓಮಾಪೋ from '../Oomapo';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');

test("ಓಮಾಪೋ", () => {
  expect(Renderer.create(
    <ಓಮಾಪೋ />
  ).toJSON()).toMatchSnapshot();

});