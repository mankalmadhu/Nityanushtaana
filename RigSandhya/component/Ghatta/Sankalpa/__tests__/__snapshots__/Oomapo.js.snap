// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಓಮಾಪೋ 1`] = `
<ವಿನಿಯೋಗ>
  ಓ॒ಮಾಪೋ॒ ಜ್ಯೋತೀ॒ರಸೋ॒ಽಮೃತ॒೦ ಬ್ರಹ್ಮ॒ ಭೂರ್ಭುವ॒ಸ್ಸ್ವರೋ೦॥
</ವಿನಿಯೋಗ>
`;
